### ISSUE TITLE
Bot keeps crashing

### ISSUE THREAD

**Issue by Terredombres** [kolton/d2bot-with-kolbot#207]
Hello,

I am totally ignorant in computer sciences, but I'd really like to bot nm, so I'm giving it a try.
I believe I followed all the instructions correctly. I made sure I have all softwares downloaded, everything went fine, until I press the button "start". There, it launches the game, but the bot crashes. I have tried with cd key selected or not, I have tried with different entry choices etc. I have read the instructions four times but can't find what I did wrong. Can you please help me?

**Comment by joekienzle** [comment]
What does it say under "console"?

**Comment by Terredombres** [comment]
Well everything crashes so it does not write anything, I have to relaunch the bot :/

**Comment by joekienzle** [comment]
Yes you need the version hack. You can install it with this instruction: https://github.com/DetectiveSquirrel/Version-Hack-Guide-1.14d/blob/master/README.md

What exactly does crash? The game or the kolbot manager? If the game crashes but the kolbot manager stays open you may find information under the tab "Console" in the kolbot manager window. But first try to install the version hacked game.

**Comment by Terredombres** [comment]
Thank you for your answer! I wonder if using a hacked version of the game makes it more likely to get detected and get the key banned? 
It was the kolbot manager that crashed, the game was launched in window format just fine.
Now it's up to me, thank you again!
